src/NECompletion-Morphic/CompletionEngine.class.st

```diff
@@ -131,13 +138,8 @@ CompletionEngine >> handleKeystrokeBefore: aKeyboardEvent editor: anEditor [
 				ifTrue: [ ^ true ] ].
 	keyCharacter = Character tab
 		ifTrue: [ 
-			NECPreferences expandPrefixes
-				ifFalse: [ 
-					menuMorph insertSelected
-						ifTrue: [ ^ true ] ]
-				ifTrue: [ 
-					menuMorph insertCommonPrefixOrSelected
-						ifTrue: [ ^ true ] ] ].
```

Comment:
The support for `insertCommonPrefixOrSelected` is removed.
1) I think it covers so few cases it is not worth it. It only expand prefixes if ALL candidates share a prefix. That is not so common. And if that does not happen, it behaves as if the setting was false...
2) the support for insertion was entirely implemented in the menu morph!